variants/frontend-base/template.rb

```diff
@@ -118,6 +118,21 @@
   EO_JS
 end
 
+prepend_to_file "app/frontend/packs/application.js" do
+  <<~'EO_JS'
+    // The application.js pack is defered by default which means that nothing imported 
+    // in this file will begin executing until after the page has loaded. This helps to 
+    // speed up page loading times, especially in apps that have large amounts of js.
+    //  
+    // If you have javascript that *must* execute before the page has finished loading, 
+    // create a separate 'boot.js' pack in the frontend/packs directory and import any 
+    // required files in that. Also remember to add a separate pack_tag entry with:
+    // <%= javascript_pack_tag "boot", "data-turbolinks-track": "reload" %> 
+    // to the views/layouts/application.html.erb file above the existing application pack tag.
+    // 
```

Comment:
```suggestion

```

🐘🐘🐘 this blank comment line doesn't add much

Reply:
Oh Its immediately followed by another comment once its all built so I wanted to leave an empty line between the two of them instead of risking them run into one another.

Reply:
oohh, all good then :D